.github/workflows/ci.yml

```diff
@@ -51,8 +51,8 @@ jobs:
           path: zksync-os
       - uses: actions/checkout@v4
         with:
-          repository: matter-labs/era-evm-tester 
-          ref: zk-ee
+          repository: matter-labs/era-evm-tester
+          ref: alocascio-zk-ee
```

Comment:
Should we use zk-ee again now?

Reply:
We should before merging, we need https://github.com/matter-labs/era-evm-tester/pull/59

Reply:
Removed